<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>370</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="View">
     <attribute name="title">
      <string>View</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>601</width>
        <height>342</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AddR">
     <property name="font">
      <font>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <attribute name="title">
      <string>AddR</string>
     </attribute>
     <widget class="QLineEdit" name="to_do">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Enter the To_Do</string>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(209, 209, 209);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32761</number>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>This field can't be empty.</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Result">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>305</y>
        <width>335</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Result</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>SizeVerCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Pick the time</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="time">
       <time>
        <hour>17</hour>
        <minute>45</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
     <widget class="QLabel" name="label_To_Do">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>40</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>false</underline>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>To Do</string>
      </property>
      <property name="text">
       <string>To Do :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_OK">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>OK</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_When">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>false</underline>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>When</string>
      </property>
      <property name="text">
       <string>When :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Pick the date</string>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximumDate">
       <date>
        <year>9999</year>
        <month>12</month>
        <day>9</day>
       </date>
      </property>
      <property name="displayFormat">
       <string extracomment="year-month-day">yyyy/M/d</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="date">
       <date>
        <year>2024</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>600</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_day">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Pick the day</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Today</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tommorow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pick a date</string>
       </property>
       <property name="icon">
        <iconset theme="accessories-calculator"/>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_Urgency">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>false</italic>
        <underline>false</underline>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Urgency</string>
      </property>
      <property name="text">
       <string>Urgency :</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_urgency">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Select the urgency</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QDial" name="dial_urgency">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>50</width>
        <height>60</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="toolTip">
       <string>Set the urgency</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="sliderPosition">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addDesc">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>135</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>false</underline>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Add Description</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(209, 209, 209);
color: rgb(0, 0, 0);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Add Description</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="to_do_Desc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>580</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="toolTip">
       <string>Description</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);
</string>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoBulletList</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_light">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30%;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/Nischal/Downloads/brightness.png</normaloff>C:/Users/Nischal/Downloads/brightness.png</iconset>
      </property>
      <property name="shortcut">
       <string>Shift+M</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_dark">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30%;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/Nischal/Downloads/night-mode.png</normaloff>C:/Users/Nischal/Downloads/night-mode.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <zorder>pushButton_light</zorder>
     <zorder>pushButton_dark</zorder>
     <zorder>to_do</zorder>
     <zorder>Result</zorder>
     <zorder>timeEdit</zorder>
     <zorder>label_To_Do</zorder>
     <zorder>pushButton_OK</zorder>
     <zorder>label_When</zorder>
     <zorder>dateEdit</zorder>
     <zorder>line</zorder>
     <zorder>comboBox_day</zorder>
     <zorder>label_Urgency</zorder>
     <zorder>spinBox_urgency</zorder>
     <zorder>dial_urgency</zorder>
     <zorder>pushButton_addDesc</zorder>
     <zorder>to_do_Desc</zorder>
    </widget>
    <widget class="QWidget" name="Settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>45</y>
        <width>551</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1.5px solid #333333;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_Urgency_2">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::HLine</enum>
       </property>
       <property name="text">
        <string>Urgency</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_SpinBox">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;</string>
       </property>
       <property name="text">
        <string>Spin Box</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Dial">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;</string>
       </property>
       <property name="text">
        <string>Dial</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>3</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1.5px solid #333333;</string>
      </property>
      <property name="text">
       <string>AddR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>135</y>
        <width>551</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1.5px solid #333333;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>3</width>
         <height>60</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_Defaultmode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;</string>
       </property>
       <property name="text">
        <string>Default mode</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Dark">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;</string>
       </property>
       <property name="text">
        <string>Dark</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Light">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;</string>
       </property>
       <property name="text">
        <string>Light</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1.5px solid #333333;</string>
      </property>
      <property name="text">
       <string>UI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_urgency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_urgency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_urgency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_urgency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
